### PR TITLE
Editor list update

### PR DESCRIPTION
**Pull request by nissimsan**
@OR13 asked that I replace him as editor.

### CODE REVIEW

**Review by mkhraisha — APPROVED** [review]

File: docs/credentials-with-issuer-dependent-terms.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "count": 0
   },
   {
-    "type": "OrganicCertificationCredential",
+    "type": "OrganicCertificateCredential",
```
Comment (TallTed) [inline docs/credentials-with-issuer-dependent-terms.json:71]: This seems out of place for this PR ... though it is certainly a change that should be made.

Comment (nissimsan) [reply docs/credentials-with-issuer-dependent-terms.json:71]: Definitely - my bad

Comment (rhofvendahl) [reply docs/credentials-with-issuer-dependent-terms.json:71]: This is due to a recent change, but does need to be merged in. Seems like it'd be fine to include this and merge.

Comment (nissimsan) [reply docs/credentials-with-issuer-dependent-terms.json:71]: OK, thanks @rhofvendahl !

**Pull request merged**